### PR TITLE
Remove myget nuget feeds

### PR DESCRIPTION
**Pull request by mavasani**

### CODE REVIEW

**Comment by mavasani** [comment]
@jmarolf Do you know what is the issue here? We are getting the below error on CI related to 2.9.x branch.

```
(NETCORE_ENGINEERING_TELEMETRY=InitializeToolset) Feed does not exist.
```

**Comment by mavasani** [comment]
Also tagging @dotpaul

**Comment by jmarolf** [comment]
myget feeds have been deleted. all branches that need to build should remove references to myget in NuGet.config.

**Comment by mavasani** [comment]
@jmarolf This PR removes the myget feed, but still seems to be hitting the missing feed issue. Should I just merge this PR?

**Comment by jmarolf** [comment]
@mavasani the eng folder also needs to be updated to not use myget I'll update this PR

**Review by jmarolf — APPROVED** [review]

**Comment by jmarolf** [comment]
@mavasani this is ready to merge

File: .editorconfig [hunk 1]
```diff
@@ -167,6 +167,9 @@ csharp_space_between_method_declaration_parameter_list_parentheses = false
 csharp_space_between_parentheses = false
 csharp_space_between_square_brackets = false
 
+# CS0618: Type or member is obsolete
+dotnet_diagnostic.CS0618.severity = none
```
Comment (mavasani) [inline .editorconfig:171]: Hmm is this required? Which obsolete member is being accessed? I am wondering if we can instead just add pragmas at specific violation sites.

Comment (jmarolf) [reply .editorconfig:171]: I can do that instead. Some test libary members were depricated

File: .editorconfig [hunk 2]
```diff
@@ -167,6 +167,7 @@ csharp_space_between_method_declaration_parameter_list_parentheses = false
 csharp_space_between_parentheses = false
 csharp_space_between_square_brackets = false
 
+
```
Comment (mavasani) [inline .editorconfig:170]:
```suggestion
```

**Comment by mavasani** [comment]
@jmarolf There are some more violations.

**Comment by jmarolf** [comment]
@mavasani I am unable to merge this

**Comment by mavasani** [comment]
Thanks a lot @jmarolf!

**Pull request merged**